webrtc/stun/stun_transport.nim

```diff
@@ -32,6 +32,12 @@ type
 
     rng: ref HmacDrbgContext
 
+proc addConnToTable(self: Stun, conn: StunConn) =
+  proc cleanup() =
+    self.connections.del(conn.raddr)
+  self.connections[conn.raddr] = conn
+  conn.addOnClose(cleanup)
```

Comment:
Then `cleanup` doesn't need to be a closure and can be passed when the conn is created. `addConnToTable` won't be necessary.